sharktank/sharktank/examples/export_paged_llm_v1.py

```diff
@@ -89,17 +90,23 @@ def main():
     else:
         model = PagedLlamaModelV1(dataset.root_theta, llama_config)
 
-    def generate_params_json(hp, prefill_bs: list[int], decode_bs: list[int]):
+    def generate_params_json(
+        hp: LlamaHParams, prefill_bs: list[int], decode_bs: list[int]
+    ) -> Dict[str, Any]:
         return {
             "module_name": "module",
             "module_abi_version": 1,
             "max_seq_len": hp.context_length,
-            "attn_head_count": hp.attention_head_count,
+            # "attn_head_count": hp.attention_head_count, # we don't need the attention head count we just need the kvcache attention head count for shortfin
```

Comment:
Since all of the docs are updated and references to "attn_head_count" are removed, this is probably fine to delete. Otherwise, it'll sit here lingering for who knows how long